index.html

```diff
@@ -0,0 +1,14 @@
+  <!DOCTYPE html>
+  <html lang="en">
```

Comment:
Required Changes:
- [x] It's better not to leave that much indentation on HTML, from now on don't leave that many empty lines because it is part of good documentation and good coding practices to keep the code clean and legible.

![image](https://user-images.githubusercontent.com/54015740/120663086-5906a400-c44f-11eb-905e-ca9c67ec45e0.png)